tests/test_model.py

```diff
@@ -1181,6 +1183,7 @@ def test_set_presentation_edition(self):
         eq_(edition_mw.is_primary_for_work, False)
         eq_(edition_od.is_primary_for_work, False)
         eq_(edition_admin.is_primary_for_work, False)
+        # TODO: does the following line need to be set to work?
```

Comment:
eq_(edition_composite.is_primary_for_work, True)
fails, because we don't tell the composite edition that it's primary for work when we set it on the pool.